README.md

```diff
@@ -12,6 +12,8 @@ deployment (the instructions for which are described in the aforementioned
 This bot simply repeats the user's query in the response and provides a good starting
 point to build any type of bot. To deploy, run `modal deploy echobot.py`
 
+A correct implementation would look like https://poe.com/EchoBotDemostration
```

Comment:
Can we fix the handle for this one? "Demonstration" instead of "Demostration"

Reply:
Edited and update the bot's handle on Poe